privacyidea/lib/tokens/pushtoken.py

```diff
@@ -860,9 +860,11 @@ def create_challenge(self, transactionid=None, options=None):
                 db_challenge.save()
                 self.challenge_janitor()
 
-            # If sending the Push message failed, we still raise an error.
+            # If sending the Push message failed, we still raise an error and a warning.
             if not res:
-                raise ValidateError("Failed to submit message to firebase service.")
+                log.warning("Failed to submit message to firebase service for token {0!s}."
+                            .format(self.token.serial))
+                raise ValueError("Failed to submit message to firebase service.")
```

Comment:
Why did you change this to ValueError?

Reply:
This really shouldn't be here because I already caught that one. I will fix it immediately.